tests/integration/telemetry/stats/prometheus/testdata/metadata_exchange_filter.yaml

```diff
@@ -14,7 +14,7 @@ spec:
       patch:
         operation: INSERT_BEFORE
         value:
-          name: envoy.wasm
+          name: envoy.filters.http.wasm
```

Comment:
I realize you are not making this change but it was made in istio/proxy at some time, but how is this supposed to work? how can a user upgrade ?

Reply:
I don't think this is exposed in any release yet. @PiotrSikora @jplevyak to double check.